content/articles/shell-script-to-demonstrate-the-se-of-shell-function-library/index.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /shell-script-to-demonstrate-the-se-of-shell-function-library/
+title: Shell Script to Demonstrate the Use of Shell Function Library
+description: This article will guide the reader what a script is, how to create one using the open script, and then demonstrate the use of the Shell Function Library using Bash scripts in Linux distributions.
+author: 
+date: 2021-08-18T00:00:00-17:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/shell-script-to-demonstrate-the-se-of-shell-function-library/hero.jpg
+    alt: Shell Script Shell Function Library
+---
+A good computer program has a lot of code in it. These code snippets can sometimes be repeated and used multiple times in the program. To simplify the process and reduce errors, functions are used to group pieces of code that are used repeatedly.

```

Comment:
Instead of `functions are used to group pieces of code that are used repeatedly` say `functions are used to group repetitive pieces of code.`